homestar-wasm/fixtures/example_add_component.wat

```diff
@@ -3432,7 +3432,7 @@
                     return
                   end
                   i32.const 0
-                  i32.load8_u offset=1049029
+                  i32.load8_u offset=1049028
```

Comment:
Purely out of curiosity: why did the offsets change/swap?

Reply:
wasm-bindgen update :).